### PR TITLE
Store Searching Preferences. #74

### PR DESCRIPTION
**Pull request by OussamaJmili**
#74 Store Searching Preferences.

* Added the Set searching preferences button
* The user can choose his searching Preferences for Job Type, Industry/Sector, Experience Level, and Remote Work Options.
* Preferences are saved for every user (stored in DB).

### CODE REVIEW

File: linkedin-clone/src/components/SearchingPreferences.js [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+        });
+    } else {
+      console.log("NO USER");
+    }
+  });
+
+  return () => {
+    unsubscribe();
+  };
+}, []);
```
Comment (TNT05) [inline linkedin-clone/src/components/SearchingPreferences.js:255]: clean

File: linkedin-clone/src/components/SearchingPreferences.js [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+<option value="science">Science</option>
+<option value="security">Security</option>
+<option value="social media">Social Media</option>
+<option value="software development">Software Development</option>
+<option value="sports">Sports</option>
+<option value="telecom">Telecom</option>
+<option value="telecommunications">Telecommunications</option>
+<option value="transportation">Transportation</option>
+<option value="travel">Travel</option>
+<option value="other">Other</option>
```
Comment (TNT05) [inline linkedin-clone/src/components/SearchingPreferences.js:339]: does the job have those possible fields as well

File: linkedin-clone/src/components/SearchingPreferences.js [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+      console.error('Error clearing preferences:', error);
+    }
+  };
+//   const handleLocationSelect = (event) => {
+//     setPreferences({
+//       ...preferences,
+//       location: event.label,
+//     });
+//   };
+// console.log(db)
```
Comment (TNT05) [inline linkedin-clone/src/components/SearchingPreferences.js:174]: we will need to cleanup the logs

**Review by TNT05 — APPROVED** [review]
clean

**Review by drbladz — APPROVED** [review]

**Pull request merged**